statsd/tests.py

```diff
@@ -8,9 +8,11 @@
 
 from statsd import StatsClient
 from statsd import TCPStatsClient
+from statsd import UnixSocketStatsClient
 
 
 ADDR = (socket.gethostbyname('localhost'), 8125)
+SOCKET = 'tmp.socket'
 
 
 # proto specific methods to get the socket method to send data
```

Comment:
rather than (confusingly) using `'tcp'` in all the unix socket tests, i'd rather add a `'unix'` option here, even if it's identical to the `'tcp'` version

```python
    'unix': lambda x: x.sendall,
```